src/Test/TestGameEvent.java

```diff
@@ -296,9 +305,12 @@ private boolean testUnitDieEvent()
     // Sink the boat. All units it holds should also be accounted for.
     GameEventQueue geq = new GameEventQueue();
     Utils.enqueueDeathEvent(boat, geq);
-    testPassed &= validate(geq.size() == 3, "    Utils.enqueueDeathEvent built the wrong number of events!");
+    // One event for each of the 3 units, and also one for losing
+    testPassed &= validate(geq.size() == 4, "    Utils.enqueueDeathEvent built the wrong number of events!");
 
-    // No cleanup required.
+    new UnitDieEvent(man).performEvent(testMap);
+    new UnitDieEvent(car).performEvent(testMap);
+    new UnitDieEvent(boat).performEvent(testMap);
```

Comment:
Why create new events instead of just executing the ones in the queue? Not that it matters, I guess.

Reply:
Didn't feel like having a CO defeat event in there.

Reply:
mumble mumble, comments for non-obvious intent, mumble.

Reply:
I put a comment in there talking about how the defeat happens...